Gemfile

```diff
@@ -5,7 +5,7 @@ source 'https://rubygems.org'
 group :release do
   gem 'emeril'
   gem 'rake'
-  gem 'chef', '~> 12.22'
+  gem 'chef', '= 14.10.9'
   gem 'foodcritic'
```

Comment:
The CI runs with foodcritic 11.4.0, and installing latest actually gives a deprecation warning. I think we should pin that version here as well.

Reply:
Done.